src/system/Form/RadioBoxGroup.stories.jsx

```diff
@@ -11,6 +11,36 @@ import { RadioBoxGroup } from '..';
 export default {
 	title: 'RadioBoxGroup',
 	component: RadioBoxGroup,
+	parameters: {
+		docs: {
+			description: {
+				component: `
+A radio-box-group is a group of radio buttons that are styled as boxes. This component is used
+to allow users to select one option from a list of options.
+
+## Known issues
+
+- The <code>chip</code> variant of the current implementation of this component does not show the
```

Comment:
I don't believe it should be considered a known issue if it's not on the design.